demos/general/demo_pipeline_voltage_imaging.py

```diff
@@ -105,55 +105,75 @@ def main():
     # first we create a motion correction object with the specified parameters
     mc = MotionCorrect(fnames, dview=dview, **opts.get_group('motion'))
     # Run correction
-    mc.motion_correct(save_movie=True)
+    do_motion_correction = True
+    if do_motion_correction:
+        mc.motion_correct(save_movie=True)
+    else: 
+        mc_list = [file for file in os.listdir(file_dir) if (fnames.split('/')[-1].split('.')[0]+'.' in file and '.mmap' in file)]
```

Comment:
us os.path.split() or some other part of the os.path library rather than looking for /
Also, doing this as a one liner makes it kind of hard to read.